lanebreach-api/lib/tasks/auto_annotate_models.rake

```diff
@@ -0,0 +1,54 @@
+# NOTE: only doing this in development as some production environments (Heroku)
+# NOTE: are sensitive to local FS writes, and besides -- it's just not proper
+# NOTE: to have a dev-mode tool do its thing in production.
+if Rails.env.development?
+  require 'annotate'
```

Comment:
This is a cool gem. I've always resorted to checking out `schema.rb` whenever I needed to check what columns exist.